tests/test_bilstm.py

```diff
@@ -43,3 +43,23 @@ def test_multilabel():
     model.fit(X, Y)
     assert model.score(X, Y) > 0.4
     assert model.predict(X).shape == (5, 4)
+
+
+def test_attention():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0, 0, 1, 1])
+
+    model = Pipeline([
+        ('vec', KerasVectorizer()),
+        ('clf', BiLSTMClassifier(
+                    nb_epochs=10,
+                    attention=True,
+                    attention_heads=10))
+    ])
+    model.fit(X, Y)
+    assert model.score(X, Y) > 0.6
```

Comment:
Wondering if this `0.6` is going to annoy our git pipelines in the future? We do `0.4` for other cases.